keyboards/kbdfans/kbd4x/kbd4x.h

```diff
@@ -21,25 +21,38 @@
 	k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, \
 	k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, \
 	k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, \
-	k30, k31, k32, k33, k34,    k35,   k37, k38, k39, k3a, k3b \
+	k30, k31, k32, k33, k34,    k35,   k37, k38, k39, k3a, k3b  \
 ) \
 { \
 	{ k00, k01, k02, k03, k04, k05, k06,   k07, k08, k09, k0a, k0b }, \
 	{ k10, k11, k12, k13, k14, k15, k16,   k17, k18, k19, k1a, k1b }, \
 	{ k20, k21, k22, k23, k24, k25, k26,   k27, k28, k29, k2a, k2b }, \
-	{ k30, k31, k32, k33, k34, k35, KC_NO, k37, k38, k39, k3a, k3b } \
+	{ k30, k31, k32, k33, k34, k35, KC_NO, k37, k38, k39, k3a, k3b }  \
 }
 
 // The PCB does support a grid layout, but the case does not.
 #define LAYOUT_ortho_4x12( \
 	k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, \
 	k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, \
 	k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, \
-	k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b \
+	k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b  \
 ) \
 { \
 	{ k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b }, \
 	{ k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b }, \
 	{ k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b }, \
-	{ k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b } \
+	{ k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b }  \
+}
+
+#define LAYOUT_kc_ortho_4x12( \
```

Comment:
I believe the use of layout macros which automatically prepend `KC_` is discouraged - it makes it difficult to use keycodes like `MO()` etc.

Reply:
I like it though, so it should probably stay.

Reply:
> I like it though, so it should probably stay.

:(